farmdata2_modules/fd2_tabs/fd2_config/config.html

```diff
@@ -1,41 +1,131 @@
+                selectedLaborConfig: '',
+                laborConfigs: ['Required', 'Optional', "Hidden"],
+                isSaved: false,
+                errBanner: null,
+            },
+            methods: {
+                setNewLabor(newLaborConfig) {
+                    this.selectedLaborConfig = newLaborConfig
+                },
+                saveConfig() {
+                    this.configObj.labor = this.selectedLaborConfig
+                    setConfiguration(this.configObj, this.sessionToken) 
+                        .then(() => {
+                            this.isSaved = true
+                            setTimeout(() => { 
+                                this.isSaved= false
+                            }, 3000)
 
-        getSessionToken().then((token) => {
-          setConfiguration(config, token)
-        })
-      }
+                        }) 
+                        .catch((err) => { 
+                            if (err.response) {
+                                // The request was made and the server responded with a status code
+                                // that falls out of the range of 2xx
+                                this.errBanner = true
+                                console.log('Error code that falls out of 2xx.')
+                            } else if (err.request) {
+                                // The request was made but no response was received
+                                // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
+                                // http.ClientRequest in node.js
+                                this.errBanner = true
+                                console.log('No response was received');
+                            } else {
+                                // Something happened in setting up the request that triggered an Error
+                                this.errBanner = true
+                                console.log('Error Message: ', err.message);
+                            }
+                        })
+                },
+                hideBanner(){
+                    this.errBanner = false
+                },
+            },
+            computed: {
+                configNotReady() {
+                    if(this.selectedLaborConfig == this.configObj.labor) {
+                        return true
+                    }
+                    return false
+                }
+            
+            },
+            created(){
+                getSessionToken().then((response) => {
+                    this.sessionToken = response
+                })
+                .catch((err) => { 
+                    if (err.response) {
+                        // The request was made and the server responded with a status code
+                        // that falls out of the range of 2xx
+                        this.errBanner = true
+                        console.log('Error code that falls out of 2xx.')
```

Comment:
These console.log messages are not necessary.   If there is a persistent error the dev tools can provide the necessary information.